rust/ql/test/library-tests/dataflow/sources/InlineFlow.expected

```diff
@@ -0,0 +1 @@
+
```

Comment:
This change can be reverted.

Reply:
hmm, weird, might be an artifact of the accepting script